src/app/Browser.js

```diff
@@ -95,30 +100,54 @@ function FileItem(props) {
 
   const copyButton = (
     <IconButton
+      title="Kopiera länk"
       style={{ boxShadow: 'none' }}
       hoveredStyle={{ boxShadow: 'none' }}
       onClick={(e) => {
         e.preventDefault()
         e.stopPropagation()
         copyToClipboard(name)
       }}>
-      <FontIcon
-        className="fas fa-link fa-xs"
-      >
-      </FontIcon>
+      <LinkIcon hoverColor="#ff5722" />
     </IconButton>
   )
 
+  
+  const isPublic = tags
+                  .filter(t => t["Key"] === "public")
+                  .map(t => t["Value"])[0]
+                  .toLowerCase() === "true"
+  
+  const AccessibleIcon = (
+    <IconButton
+      style={{ boxShadow: 'none' }}
+      hoveredStyle={{ boxShadow: 'none' }}
+      title={isPublic ? "Öppen för alla på internet" : "Ej öppen för alla på internet, åtkomst via yoggi krävs."}
+      onClick={e => {
+        e.preventDefault()
+        e.stopPropagation()
+        onPublicSet(name, isPublic)
+      }}
+    >
+      {isPublic ?
+        <LockOpen hoverColor="#ff5722" />
+        :
+        <LockClosed hoverColor="#ff5722" />
+      }
+    </IconButton>
+  )
 
   return (
-    <a href={`/${name}`}> <ListItem
-      primaryText={name}
-      rightIconButton={
-        <div>
-          {copyButton}
-          {deleteButton}
-        </div>}
-    />
+    <a href={isPublic ? `https://${process.env.REACT_APP_BUCKET_NAME}.s3.amazonaws.com/${name}` : `/${name}`} target="_blank" rel="noopener noreferrer">
```

Comment:
Does this solve links being https as well?

Reply:
What do you mean exactly?

Reply:
The thing you wrote to me about before. That pressing the copy link-button (?) created HTTP links.

Reply:
Ahh, I fixed it in a3136134607f5ff6ad217a7df1091fb4ba0678b5

`window.location.host` copies the path without protocol, for example `github.com/datasektionen`
`window.location.origin` copies the path the protocol, for example `https://github.com/datasektionen`